### PR TITLE
MF-661 - Bootstrap pagination in UI

### LINKED ISSUE #661: Fix pagination
Switch from using custom pagination function to Bootstrap.Pagination module:
https://package.elm-lang.org/packages/rundis/elm-bootstrap/5.1.0/Bootstrap-Pagination

### PR DESCRIPTION
**Pull request by blokovi**
### What does this do?
Switch to bootstrap style pagination

### Which issue(s) does this PR fix/relate to?
Resolves #661

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Open the _src/Env.elm_ file and edit the values of the `env` record.
 
 Follow the instructions above to install and run GUI as a native build. In
 _src/Env.elm_ change a `url` field value of the `elm` record to
-`http://localhost:80`. Instead of `make run` you can install `elm-live`
+`http://localhost:80/`. Instead of `make run` you can install `elm-live`
```
Comment (manuio) [inline ui/README.md]: why that?

**Review by manuio — CHANGES_REQUESTED** [review]

File: ui/README.md (hunk 1 above)
Comment (blokovi) [reply ui/README.md]: UI doesn't work without trailing slash in this variable. I found this error in README while working on this issue but it's not related to pagination. Should I make seperate NOISSUE PR for this?

Comment (drasko) [reply ui/README.md]: Nope, here is OK. Just that this must be noted somewhere.

Comment (manuio) [reply ui/README.md]: ok, I resolve this one then

**Review by manuio — DISMISSED** [review]

File: ui/README.md (hunk 1 above)
Comment (drasko) [reply ui/README.md]: Make a special note that trailing `/` is mandatory, as it is often source of errors.

> You can use markdown quote like this for this kind of notices.

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**